doc/dev_start_guide.txt

```diff
@@ -267,41 +252,37 @@ You can use ``easy_install`` instead of ``pip``, and ``pep8`` instead of
 ``flake8`` if you prefer. The important thing is that the ``flake8`` or
 ``pep8`` executable ends up in your ``$PATH``.
 
-To install Syntastic, according to its documentation, the easiest way is
-to install `pathogen.vim`_ first.
-
-Here's a relevant extract of pathogen.vim's installation instructions:
-
-    Install to ``~/.vim/autoload/pathogen.vim``. Or copy and paste::
-
-        mkdir -p ~/.vim/autoload ~/.vim/bundle; \
-        curl -so ~/.vim/autoload/pathogen.vim \
-            https://raw.github.com/tpope/vim-pathogen/HEAD/autoload/pathogen.vim
+Install vundle with: ``git clone https://github.com/VundleVim/Vundle.vim.git
```

Comment:
Let's make these instructions a numbered list:

``` rest
To setup VIM:

#. Install flake8 (if not already installed) with::

    pip install flake8

   .. note:: You can use ``easy_install`` instead of ``pip``, and ``pep8`` 
      instead of ``flake8`` if you prefer. The important thing is that the 
      ``flake8`` or ``pep8`` executable ends up in your ``$PATH``.

#. Install vundle with:

    git clone https://github.com/VundleVim/Vundle.vim.git ~/.vim/bundle/Vundle.vim

#. Edit `~/.vimrc` and add the lines::

    .. code-block:: python

    set nocompatible              " be iMproved, required
    filetype off                  " required
    " set the runtime path to include Vundle and initialize
    set rtp+=~/.vim/bundle/Vundle.vim
    call vundle#begin()

    Plugin 'gmarik/Vundle.vim' " let Vundle manage Vundle (required!)
    Plugin 'scrooloose/syntastic'
    Plugin 'jimf/vim-pep8-text-width'

    " Syntastic settings
    " You can run checkers explicitly by calling :SyntasticCheck <checker
    let g:syntastic_python_checkers = ['flake8'] "use one of the following checkers:
                                                 " flake8, pyflakes, pylint, python (native checker)
    let g:syntastic_enable_highlighting = 1  "highlight errors and warnings
    let g:syntastic_style_error_symbol = ">>" "error symbol
    let g:syntastic_warning_symbol = ">>" "warning symbol
    let g:syntastic_check_on_open = 1
    let g:syntastic_auto_jump = 0  "do not jump to errors when detected

#. Open a new vim and run ``:PluginInstall`` to automatically install the
   plugins. When the installation is done, close the installation "window" with ``:q``. From now on Vim will check for PEP8 errors and highlight them whenever a file is saved.

A few useful commands
"""""""""""""""""""""
```
Note that the first line of `.. note:: ` is indented at the same level as `To install flake8`. Every other line is indented as the `note` word.